packages/observe-sequence/observe_sequence.js

```diff
@@ -140,14 +140,55 @@ ObserveSequence = {
     } else if (isStoreCursor(seq)) {
       return seq.fetch();
     } else {
-      throw badSequenceError();
+      throw badSequenceError(seq);
     }
   }
 };
 
-var badSequenceError = function () {
+var ellipsis = function (longStr, maxLength) {
```

Comment:
As we are adding new code here we should use new patterns, like const here instead of var.

Also everywhere below this point.

Reply:
I usually try to stick to the existing coding style, but you're right